modules/react/pagination/lib/Pagination/GoTo/Label.tsx

```diff
@@ -7,6 +7,9 @@ import {PaginationModel} from '../types';
 
 export interface GoToLabelProps extends React.LabelHTMLAttributes<HTMLLabelElement> {
   model: PaginationModel;
+  /**
+   * Accepts child elements or a render prop.
+   */
   children?: (model: PaginationModel) => React.ReactNode | React.ReactNode;
```

Comment:
I had to add a JSDoc comment for `children` in order for it to show up in `<ArgsTable of={Pagination.GoToLabel} />`. May have something to do with [this](https://github.com/storybookjs/storybook/issues/9921#issuecomment-655436418).